### DISCUSSION TITLE
upper icons on Right side of screen

### DISCUSSION THREAD

**Discussion by ghost** [elementary/wingpanel#374] [Q&A] [answered]
I am new to EOS and I have a question that might of been answered or is somewhere on the net but I cannot seem to find it.  In the upper right of the screen that has the speaker settings, internet and such is is possible to add a bar that would show active background apps that might be running.  I have a couple of apps that I like to run and deactivate at times.  One is Caffeine that lets you keep the screen on and the other is Variety that adds a big clock to the desktop and changes your wallpaper at intervals.   
Again I am sorry if this has been asked and if this is not really the area to ask it.  I love EOS and plan on using and supporting for a long time.
John

**Comment by ryonakano** [comment, marked as answer]
Unfortunately we no longer support app indicators; it seems to be deprecated in GNOME too and there are multiple alternative way that the app can notify users the app is running or some action is in progress. See the previous discussions here: https://github.com/elementary/wingpanel/discussions/360 and https://github.com/elementary/wingpanel/issues/96

**Comment by ghost** [comment]
Thank you for the quick reply and help
